### PR TITLE
Only good examples in the gallery

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

File: examples/trellis_barley.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "description": "Classic Trellis Example.",
+  "description": "Classic Barleys Example. Shows barley yields for diffent years across varieties and sites.",
```
Comment (kanitw) [inline examples/trellis_barley.json]: Why "classic barley example”?  This is a classic trellis plot in the paper by Becker and Cleveland.  No one knows it by the name “Barley” except ones who are familiar with Vega examples.

Comment (kanitw) [inline examples/trellis_barley.json]: Also typo “diffent"

File: examples/vlexamples.json [hunk 2]
```diff
@@ -2,23 +2,28 @@
...
+      "showInEditor": true
     },
     {
       "name": "scatter",
-      "title": "A scatterplot"
+      "title": "A scatterplot",
+      "showInEditor": true
     },
     {
       "name": "scatter_binned_axes",
```
Comment (kanitw) [inline examples/vlexamples.json]: This one is good too?

File: site/gallery.js [hunk 3]
```diff
@@ -1,45 +1,90 @@
 'use strict';
 
-/* global vl, d3, vg */
+/* global vl, d3, vg, console, location, window, document */
 
 d3.select('#vl-version').text(vl.version);
 d3.select('#vg-version').text(vg.version);
 
+var params = function() {
```
Comment (kanitw) [inline site/gallery.js]: Add comment that this method parse url parameter?

**Comment by kanitw** [comment]
LGTM besides the comment I have .. feel free to merge after fixing.

File: examples/trellis_barley.json (hunk 1 above)
Comment (domoritz) [inline examples/trellis_barley.json]: Done

File: examples/vlexamples.json (hunk 2 above)
Comment (domoritz) [inline examples/vlexamples.json]: Done

**Pull request merged**